packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx

```diff
@@ -50,18 +52,7 @@ async function fetchBalanceDetailInfo(
 const WalletBalanceContainer: React.FC = () => {
   const [connected, setConnected] = useState(true);
   const [address, setAddress] = useState("");
-  const [width, setWidth] = useState(Number);
-  const handleResize = () => {
-    setWidth(window.innerWidth);
-  };
-
-  useEffect(() => {
-    handleResize();
-    window.addEventListener("resize", handleResize);
-    return () => {
-      window.removeEventListener("resize", handleResize);
-    };
-  }, []);
+  const [breakpoint] = useAtom(CommonState.breakpoint);
```

Comment:
Why didn't you use a variable in `useWindowSize` hook?

Reply:
I checked the design and I don't think I'll need it. Thanks!

Reply:
In the WalletBalanceSummary component, the width is changed at the Mobile breakpoint and is being used.